website/pages/en/cookbook/subgraph-debug-forking.mdx

```diff
@@ -44,12 +44,12 @@ export function handleUpdatedGravatar(event: UpdatedGravatar): void {
 }
 ```
 
-Oops, how unfortunate, when I deploy my perfect looking subgraph to the [Hosted Service](https://thegraph.com/hosted-service/) it fails with the _"Gravatar not found!"_ error.
+Oops, how unfortunate, when I deploy my perfect looking subgraph to the [hosted service](https://thegraph.com/hosted-service/) it fails with the _"Gravatar not found!"_ error.
```

Comment:
There are quite a few edits which are capitalisation changes?
I think we need to update examples etc. to use the Studio and Network

Reply:
My thought process here is that we can change the hosted service capitalization now, and then when Sunbeam starts, we can replace hosted service with studio. Afaik, we are entering something of a liminal space between Sunray's completion and the Sunbeam phase, so I'd imagine we should keep hosted service in docs for now?